components/neeva/browser/content_filtering_service.cc

```diff
@@ -22,12 +22,13 @@ ContentFilteringService::ContentFilteringService(int render_process_id)
 ContentFilteringService::~ContentFilteringService() = default;
 
 // static
+// Creates the Service. Called in weblayer/browser/content_browser_client_impl.cc
```

Comment:
Moving this comment to inside the function then

Reply:
Actually just removing this function. This one is easy to find in the codebase because you can do a search for `ContentFilteringService::AddInterface(`